extensions/amp-story/1.0/amp-story.js

```diff
@@ -212,8 +212,10 @@ export class AmpStory extends AMP.BaseElement {
     /** @private @const {!UnsupportedBrowserLayer} */
     this.unsupportedBrowserLayer_ = new UnsupportedBrowserLayer(this.win);
 
-    /** Instantiates the viewport warning layer. */
-    new ViewportWarningLayer(this.win, this.element);
+    if (!isExperimentOn(this.win, 'disable-amp-story-desktop')) {
```

Comment:
This allows mobile landscape though, we need to keep it in mind if we ever allow launching this experiment

Reply:
Ah, good point; I will see if I can fix that

Reply:
Updated; PTAL